src/ui/components/Alert/Alert.tsx

```diff
@@ -56,7 +56,6 @@ const Alert = ({
       }`}
     >
       <IonAlert
-        data-testid={dataTestId}
```

Comment:
Because we added this test id on parent of `IonAlert` then I think it's not necessary here. 
And It also cause an error in test case when we find 2 elements with same test id.

Reply:
Thanks for clarifying. Can we change the parent to something like: `data-testid={dataTestId + "-container"}` and bring back the test id for the IonAlert so they are different? I think Jakub needs it.